src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -120,15 +121,23 @@ private void reportTelemetry() {
             telemetry.property(OPENSHIFT_VERSION, info.getOpenshiftVersion());
             telemetry.send();
         } catch (RuntimeException e) {
-            //workaround to not send null values
-            if (e.getMessage() != null) {
-                telemetry.error(e).send();
-            } else {
-                telemetry.error(e.toString()).send();
+            // do not send telemetry when there is no context ( ie default kube URL as master URL )
+            if (!e.getMessage().startsWith(Constants.DEFAULT_KUBE_URL)){
+                //workaround to not send null values
+                if (e.getMessage() != null) {
+                    telemetry.error(e).send();
+                } else {
+                    telemetry.error(e.toString()).send();
+                }
             }
         }
     }
 
+    @Override
+    public boolean isOpenshift(){
```

Comment:
Rename to isOpenShift to respect branding

Reply:
And the existing method should have been updated rather the deleted and new method created, that makes readers life much easier